app/db/db.py

```diff
@@ -0,0 +1,16 @@
+import psycopg2 
+con = psycopg2.connect(dbname='ride', user='postgres', host='localhost', password='python')
+
+def insertUser(data):
+	fname,lname,location = data['fname'],data['lname'],data['location']
```

Comment:
Check on your naming convention; something like full_name would be nice and readable.